sendsoknad-web/src/main/java/no/nav/sbl/dialogarena/rest/ressurser/VedleggRessurs.java

```diff
@@ -230,20 +245,28 @@ private Vedlegg lagVedlegg(String behandlingsId, Vedlegg forventning, long sokna
                 .medFillagerReferanse(forventning.getFillagerReferanse())
                 .medOpprettetDato(forventning.getOpprettetDato())
                 .medInnsendingsvalg(UnderBehandling)
-                .medAntallSider(PdfUtilities.finnAntallSider(file));
-        vedlegg.setFilnavn(returnerFilnavnMedFiltype(behandlingsId, vedlegg, file));
+                .medAntallSider(antallSider);
+        vedlegg.setFilnavn(returnerFilnavnMedFiltype(behandlingsId, vedlegg, file, antallSider));
         return vedlegg;
     }
 
-    private String returnerFilnavnMedFiltype(String behandlingsId, Vedlegg vedlegg, byte[] file) {
-        boolean erPdfa = PdfUtilities.erPDFA(behandlingsId, file);
-
+    private String returnerFilnavnMedFiltype(String behandlingsId, Vedlegg vedlegg, byte[] file, int antallSider) {
         String filnavn = vedlegg.lagFilNavn();
+        logger.info("{}: Start sjekk erPDFA", behandlingsId);
+        boolean erPdfa = sjekkOmPdfa(behandlingsId, vedlegg, file, antallSider);
+        logger.info("{}: End sjekk erPDFA", behandlingsId);
+
         filnavn = StringUtils.removeEnd(filnavn, ".pdf");
         filnavn = StringUtils.removeEnd(filnavn, ".pdfa");
         return filnavn + (erPdfa ? ".pdfa" : ".pdf");
     }
 
+    private boolean sjekkOmPdfa(String behandlingsId, Vedlegg vedlegg, byte[] file, int antallSider) {
+        if (antallSider <= 2) return PdfUtilities.erPDFA(behandlingsId, file);
```

Comment:
Endre til noe litt høyere? 10 eller eventuelt mer
```suggestion
        if (antallSider <= 10) return PdfUtilities.erPDFA(behandlingsId, file);
```

Reply:
Endret til 10, og flytter sjekk av om kvitteringsvedlegg foran sjekk om PDFA.